clients/client-transcribe-streaming/test/index.integ.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { TranscribeStreaming } from "../index";
+import { createReadStream } from "fs";
+import { join } from "path";
+const moduleRoot = join(__dirname, "..", "..", "..");
+const audio = createReadStream(join(moduleRoot, "test", "speech.wav"));
```

Comment:
This is a workaround because the test file will be moved to `dist` folder when compiled to JS but `speech.wav` is left out of the folder.

Reply:
Addressed in 4c7bff076e884847ceb76a780d7e5074357ea2b1. I added a postbuild script `cp speech.wav`  to `dist` folder.

Reply:
> This is a workaround because the test file will be moved to dist folder when compiled to JS but speech.wav is left out of the folder.

We should move to prioritize moving to ts-jest to avoid these shenanigans